scvi/module/_jaxvae.py

```diff
@@ -73,6 +73,9 @@ def __call__(self, z: jnp.ndarray, batch: jnp.ndarray, is_training: bool):
         )
         h = nn.relu(h)
         h = nn.Dropout(self.dropout_rate)(h, deterministic=not is_training)
+        if h.ndim == 3:
```

Comment:
this is an interesting choice haha.. wonder if there is a more explicit way to bake this into the decoder

Reply:
@PierreBoyeau  let's instead do the same thing I do at line 69, as this is something that can be done with jax but not pytorch efficiently